5eShaped/translations/it.json

```diff
@@ -835,9 +835,9 @@
   "SIZE": "Taglia",
   "SKILL": "Abilità",
   "SKILLS": "Abilità",
-  "SKILL_$0": "skill_$0",
-  "SKILL_$1": "skill_$1",
-  "SKILL_$10": "skill_$10",
+  "SKILL_$0": "Abilità_$0",
```

Comment:
Hmm, these actually shouldn't even be translated as they are the name of the field.. I'll fix that on my end